src/main/java/com/yurjinia/common/security/jwt/filters/JwtAuthenticationFilter.java

```diff
@@ -47,7 +48,7 @@ protected void doFilterInternal(@NonNull HttpServletRequest request,
         } catch (CommonException ex) {
             handleCommonException(response, ex);
         } catch (Exception ex) {
-            handleGeneralException(response, ex);
+            handleCommonException(response, new CommonException(ErrorCode.FORBIDDEN, HttpStatus.FORBIDDEN, List.of(ex.getMessage())));
```

Comment:
I think in case we catch an exception that is not related to our project, but something completely different - it's better to throw 500 Internal Server Error exception. `Forbidden` is more about not allowing to execute an endpoint by user that doesn't have the appropriate rights.